src/components/FontAwesomeIcon.js

```diff
@@ -12,7 +18,7 @@ function normalizeIconArgs (icon) {
     return null
   }
 
-  if (typeof icon === 'object' && icon.prefix && icon.iconName) {
+  if (icon && typeof icon === 'object' && icon.prefix && icon.iconName) {
```

Comment:
I see that `icon.icon` is the only difference here between current line 21 and line 9 change. 
Would we ever not have an `icon.icon` property?
I see that this property is the icon details, unicode, svg, etc.  My only thought here is that this covers webfonts in the case of an icon not using an svg ... is this correct, so we want to return the icon even if NOT an svg ?